### PR TITLE
[0.9.x][KOGITO-1774] - Update Jobs-services, data-index and mgmt cons…

### PR DESCRIPTION
**Pull request by spolti**
…ole SNAPSHOT artefacts

jobs-services snapshot version: jobs-service-0.9.1-20200409.104441-9-runner.jar
md5: d8559ab6f897436918ea5f150a7b9b10

data-index snapshot version: data-index-service-0.9.1-20200409.104935-9-runner.jar
md5: a7f38c232405f23fddc73c617b5a8307

management-console snapshot version: management-console-0.9.1-20200409.105335-9-runner.jar
md5: 9cbbffb033913cfc9fe8c37dbfc427cf

Image tag: 0.9.1-rc1

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

File: tests/features/kogito-quarkus-jvm-ubi8.feature [hunk 1]
```diff
@@ -30,7 +30,7 @@ Feature: Kogito-quarkus-ubi8 feature.
       | path            | /hello                   |
       | wait            | 80                       |
       | expected_phrase | Mario is older than Mark |
-    And file /home/kogito/bin/drools-quarkus-example-0.9.0-runner.jar should exist
+    And file /home/kogito/bin/drools-quarkus-example-8.0.0-SNAPSHOT-runner.jar should exist
```
Comment (radtriste) [inline tests/features/kogito-quarkus-jvm-ubi8.feature]: version should be `0.9.1-SNAPSHOT`

File: tests/features/kogito-quarkus-ubi8-s2i.feature [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Feature: kogito-quarkus-ubi8-s2i image tests
 
   Scenario: Verify if the s2i build is finished as expected and if it is listening on the expected port
-    Given s2i build /tmp/kogito-examples from drools-quarkus-example using 0.9.0 and runtime-image quay.io/kiegroup/kogito-quarkus-ubi8:latest
+    Given s2i build /tmp/kogito-examples from drools-quarkus-example using master and runtime-image quay.io/kiegroup/kogito-quarkus-ubi8:latest
```
Comment (radtriste) [inline tests/features/kogito-quarkus-ubi8-s2i.feature]: `0.9.x` instead of `master` ?

**Review by radtriste — CHANGES_REQUESTED** [review]
see my comments for the tests

**Comment by radtriste** [comment]
I would remove the label `cherry-pick` as it is not here

File: tests/features/kogito-quarkus-ubi8-s2i.feature (hunk 2 above)
Comment (spolti) [reply tests/features/kogito-quarkus-ubi8-s2i.feature]: ahm yeah, those was automatically changed by the version-management script.
will fix in a few minutes.

**Review by ricardozanini — CHANGES_REQUESTED** [review]
There's a couple 8.0.0-SNAPSHOT that should be 0.9.1-SNAPSHOT

File: tests/features/kogito-quarkus-jvm-ubi8.feature (hunk 1 above)
Comment (spolti) [reply tests/features/kogito-quarkus-jvm-ubi8.feature]: done

File: tests/features/kogito-quarkus-ubi8-s2i.feature (hunk 2 above)
Comment (spolti) [reply tests/features/kogito-quarkus-ubi8-s2i.feature]: done

**Comment by spolti** [comment]
changes applied, behave tests passed locally.

```
1 feature passed, 0 failed, 8 skipped
4 scenarios passed, 0 failed, 47 skipped
16 steps passed, 0 failed, 208 skipped, 0 undefined
Took 1m6.439s
2020-04-09 20:00:53,189 cekit        INFO     Finished!
```

**Comment by radtriste** [comment]
can you update the title with correct issue ? (KOGITO-1774 ;)

**Review by sutaakar — APPROVED** [review]

**Comment by radtriste** [comment]
tests were good

**Comment by spolti** [comment]
@radtriste good catch hehe
updated.

**Pull request merged**